addons/utm/views/utm_campaign_views.xml

```diff
@@ -81,6 +84,7 @@
                 <field name='color'/>
                 <field name='user_id'/>
                 <field name="stage_id"/>
+                <field name='active'/>
                 <templates>
                     <t t-name="kanban-menu">
```

Comment:
Doesn't it create that menu automatically if not specified?

Maybe we want a custom one here, not sure, would you mind checking?

Reply:
That is fine then, I thought that the framework auto-generated this menu but maybe it's an old behavior.

Thank you for the detailed explanation :smile: